compose.prod.yml

```
@@ -49,16 +45,13 @@ services:
     networks:
       - webserver
 
-  php:
-    build:
-      context: .
-      dockerfile: ./docker/php/Dockerfile
+  api:
```

```suggestion
  php:
```

Je pense que c'est mieux de pas rename les services